ChangeLog.d/c-build-helper-hostcc.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Use HOSTCC (if it is set) when compiling C code during generation of the
+     generated files. This allows them to be generated when CC is set for
```

Comment:
“generation of the generated files” is uninformative. In contexts like this we call them “[generated] configuration-independent files”.